src/project-css/app/Http/Controllers/DataViewController.php

```diff
@@ -55,27 +62,28 @@ public function index(Request $request, $curTable){
                             ->with('tblId',$curTable);
   }
 
+  /**
+  * Show a record in the table
+  */
   public function show(Request $request, $curTable, $curId){
```

Comment:
$request parameter is not used and can be removed.

Reply:
removed